pkg/dockerfilegen/dockerfile-templates/FuncUtil.dockerfile.tmpl

```diff
@@ -33,15 +33,17 @@ RUN ln -s {{.app_file}} /usr/local/bin/deploy && \
 USER 65532
 
 LABEL \
-      com.redhat.component="openshift-serverless-1-{{.project_dashcase}}{{.component_dashcase}}-{{.rhelVersion}}-container" \
-      name="openshift-serverless-1/{{.project_dashcase}}{{.component_dashcase}}-{{.rhelVersion}}" \
+      com.redhat.component="openshift-serverless-1-{{.name}}-container" \
+      name="openshift-serverless-1/{{.name}}" \
       version=$VERSION \
       summary="Red Hat OpenShift Serverless 1 {{.project}}{{.component}}" \
       maintainer="serverless-support@redhat.com" \
       description="Red Hat OpenShift Serverless 1 {{.project}}{{.component}}" \
       io.k8s.display-name="Red Hat OpenShift Serverless 1 {{.project}}{{.component}}" \
       io.k8s.description="Red Hat OpenShift Serverless {{.project}}{{.component}}" \
       io.openshift.tags="{{.component_dashcase}}" \
+      vendor="Red Hat, Inc." \
+      release=$VERSION \
```

Comment:
Shouldn't it be `{{.version}}` given it's template?

Reply:
Yes, good catch!